app.rb

```diff
@@ -1,8 +1,71 @@
 require 'sinatra/base'
+require './lib/rps_game'
+
 class RockPaperScissors < Sinatra::Base
-  get '/test' do
-    'test page'
+  
+  enable :sessions
+  
+  get '/' do
+    erb(:index)
+  end
+
+  post '/sign_in' do
+    session[:player1] = params[:player1]
+    session[:player2] = params[:player2]
+    redirect '/play'
+  end
+
+  get '/play' do
+    session[:game] = create_game
+    redirect 'player_1_choice'
+  end
+
+  get '/player_1_choice' do
+    @game = session[:game]
+    erb(:turn1)
+  end
+
+  post '/player_1_choice' do
+    @game = session[:game]
+    session[:player_1_choice] = params[:player_1_choice]
+    redirect '/player_2_choice' if @game.player2 != 'Computer'
+    redirect '/winner'
+  end
+
+  get '/player_2_choice' do
+    @game = session[:game]
+    erb(:turn2)
+  end
+
+  post '/player_2_choice' do
+    session[:player_2_choice] = params[:player_2_choice]
+    redirect '/winner'
+  end
+
+  get '/winner' do
+    @winner_message = define_winner
+    @winner_message += " won" unless @winner_message == 'Draw'
+    erb(:winner)
+  end
+
+  private
+
+  def create_game
```

Comment:
These methods are very neat and I can see why you've put them here, but I'm not sure that the controller file is the right place to define new methods - could you integrate these methods into your game class?